libtcmu_log.h

```diff
@@ -14,6 +14,8 @@
 #include <stdarg.h>
 #include <stdbool.h>
 
+#include "libtcmu_config.h"
```

Comment:
Just do a forward declartion for struct tcmu_config like we do for struct tcmu_device.

Reply:
Sure, will fix it.
Thanks.